crm_salesperson_planner_sales/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module allows you to manage sales from commercial visits.
```

Comment:
A bit of usage explanation?

Reply:
I have explained a little more, what do you think?